docs/Manage-Space.rst

```diff
@@ -578,117 +544,91 @@ Inviting new members
-
-   -  **The third way**
-
-      |image39|
-
-      1. |image40| Go to Members application of the space.
-
-      2. |image41| Enter the username of the person you wish to invite to
+3- Enter the username of the person you wish to invite to
       the space. You can just type in the first letters and a list of
       suggestions should appear. This list contains persons having those
       letters in their username, First name or Last name. Press Enter on
       keyboard to confirm the user selection.
 
-      If you entered a wrong username (i.e it doesn't exist), it gets underlined in red:
-      
-      |image42|
 
-      3. |image43| Choose one or more persons to invite from the list. You
-      can remove some persons by just clicking on |image10| in front of
-      the displayed named.
+.. note:: Every user, or space, added to the invite drawer, is presented by the full name and its avatar.
+         If you entered a wrong username (i.e it doesn't exist), nothing will be displayed in the suggestions list. 
 
-      4. |image44| Click on Invite to send invitations to the chosen
+4. Click on Invite to send invitations to the chosen
       persons.
 
-      If you press on Enter to confirm a wrong username and then click
-      on Invite, an error pop up appears indicating that the selected
-      username is not valid.
+      If you press on Invite to confirm a wrong username, the wrong username is not taken into account on sent invitations.
       
-      |image60|
+ 
+The invitees will see your invitations in the :ref:`Invitations application <Accept-Ignore-Space-Invitation>` at the right panel of their homepage.
 
-   -  **Inviting users from a group**
+Filter space members
+-----------------------
+You can filter space's members by choosing a value from the pulldown: 
 
-      1. Click |image45| to open the Select a Group form.
+- Members: Displays all the space's members
 
-      2. Select a group on the left pane, then select its sub-group on the right pane.
+- Managers: Displays only the space's managers
 
-      3. Click Invite to invite your selected group.
+- Invited: displays invited users to the space
 
-After that, you will see the list of invited users. The invitees will
-see your invitations in the :ref:`Invitations application <InvitationsApp>` at the right panel of their homepage.
+- Requested: displays users that send a request to join the space.
 
-Revoking your invitations
----------------------------
+.. note:: Invited and Requested are visible to only the space managers.
 
-If the invited users have not accepted your requests yet, you can revoke
-your invitations by clicking |image46| corresponding to the users' name.
-The users will be removed from the Invited list.
+Display Invited members
+-------------------------
 
-Validating/Declining request
------------------------------
+You can display the list of invited members just by selecting **Invited** on the filter space members pulldown.
```

Comment:
```suggestion
You can display the list of invited users just by selecting **Invited** on the filter space members pulldown.
```

users become members only after they are accepted. so invited members does not exist. it's invited users.